<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPort"><property name="text"><string>Port:</string></property></widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="portEdit"><property name="text"><string>9000</string></property></widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelKey"><property name="text"><string>Key:</string></property></widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="keyEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="startButton"><property name="text"><string>Start Server</string></property></widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="log">
     <property name="readOnly"><bool>true</bool></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
